src/main/java/us/spaceclouds42/zones/mixin/ServerPlayNetworkHandlerMixin.java

```diff
@@ -156,7 +156,20 @@ private void renderBorders(CallbackInfo ci) {
         }
     }
 
-    @Inject(method = "sendPacket(Lnet/minecraft/network/Packet;Lio/netty/util/concurrent/GenericFutureListener;)V", at = @At("HEAD"), cancellable = true)
+    /**
+     * Prevents packets containing cloaked blocks from sending
```

Comment:
This doesn't prevent packets from sending, this just makes them send an air block to the client.

Reply:
oh oops